lib/profiler/get_tick.h

```diff
@@ -67,6 +67,15 @@ get_tick(void)
     return regs;
 }
 
+#elif defined(HAVE_CLOCK_GETTIME_NSEC_NP)
+#include <time.h>
+
+static __inline hrtime_t
+get_tick(void)
```

Comment:
Nitpick: Squid/C++ style favors `()`

```suggestion
get_tick()
```